src/frontend/komponenter/Fagsak/Behandlingsresultat/Valutakurs/Valutakurser.tsx

```diff
@@ -44,11 +50,34 @@ interface IProps {
 
 const Valutakurser: React.FC<IProps> = ({ valutakurser, åpenBehandling, visFeilmeldinger }) => {
     const { erValutakurserGyldige } = useEøs();
+    const { toggles } = useApp();
+    const månedligValutajusteringToggleErSlåttPå = toggles[ToggleNavn.månedligValutajustering];
+
+    const finnesValutaperioderSomKanSkjules =
+        valutakurser.length > 1 && månedligValutajusteringToggleErSlåttPå;
+    const [visAlleValutaperioder, setVisAlleValutaperioder] = useState(false);
+
     return (
         <ValutakurserContainer>
-            <Heading spacing size="medium" level="3">
-                Valuta
-            </Heading>
+            <HStack gap="3" wrap={false}>
```

Comment:
Hva gjør gap og wrap her?

Reply:
Wrap er der for å unngå at knappen skal wrappe seg ved liten skjerm:
Med wrap satt til false
 

https://github.com/navikt/familie-ba-sak-frontend/assets/110383605/73f0d072-5e19-4c86-93e7-fdb15eeaa489


Uten wrap (default hstack)

https://github.com/navikt/familie-ba-sak-frontend/assets/110383605/e86692d4-b7c7-479c-b869-19c22d595d2c

Reply:
Gap er der for at vi skal ha litt mellomrom mellom knapp og label uansett hvor liten skjermen blir,  gap mellom ellementer i hstack.

Med gap

https://github.com/navikt/familie-ba-sak-frontend/assets/110383605/859a55b1-59b8-4d0c-b72b-f9118ea2af50



uten gap

https://github.com/navikt/familie-ba-sak-frontend/assets/110383605/848ba86f-498d-438e-a7c4-721d1421a769